api/resource.rb

```diff
@@ -198,10 +198,17 @@ def parameters
       (@parameters || []).reject(&:exclude)
     end
 
-    def exclude_if_not_in_version(version)
+    def below_version?(version)
+      version < min_version
+    end
+
+    # Recurses through all nested properties and paramters and changes their
+    # 'exclude' instance variable if the property is at a version below the
+    # one that is passed in.
+    def exclude_if_not_in_version!(version)
       @exclude ||= version < min_version
```

Comment:
```suggestion
       @exclude ||= below_version? version
```

We can use the new method here, right?